schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/process/translate.xsl

```diff
@@ -32,225 +32,176 @@
                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                 xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 xmlns:xs="http://www.w3.org/2001/XMLSchema"
-                xmlns:gn-fn-metadata="http://geonetwork-opensource.org/xsl/functions/metadata"
+                xmlns:geonet="http://www.fao.org/geonetwork"
```

Comment:
Is it required a template to remove `geonet` elements also, other xslt in the `process` folder has that type of template?

in the same xslt for iso19139 this line doesn't exist.

Reply:
For harvesting no. I only tested suggestion with 115-3 for now but turned it off as it requires a bit more effort. Maybe we can check later when we add translation service in the editor and also align 19139 ?

Reply:
`geonet` namespace is required for suggestion templates